3rd-party-licenses.md

```diff
@@ -0,0 +1,147 @@
+# License overview of included 3rd party libraries
+
+Shopsys Framework is licensed under the terms of the [Shopsys Community License](./LICENSE).
+
+Shopsys Framework also uses some third-party components and images
+which are licensed under their own respective licenses.
+
+## Components used by Shopsys Framework
```

Comment:
Is the list in any particular order? I would sort them alphabetically for easier lookup.